test/integrations/destinations/google_adwords_remarketing_lists/router/data.ts

```diff
@@ -573,4 +573,408 @@ export const data = [
+                  typeOfList: 'General',
+                },
+              },
+              message: {
+                action: 'update',
+                context: {
+                  ip: '14.5.67.21',
+                  library: {
+                    name: 'http',
+                  },
+                },
+                recordId: '2',
+                rudderId: '2',
+                fields: {
+                  email: 'test@abc.com',
+                  phone: '@09876543210',
+                  firstName: 'test',
+                  lastName: 'rudderlabs',
+                  country: 'US',
+                  postalCode: '1245',
+                },
+                type: 'record',
+              },
+              metadata: {
+                secret: {
+                  access_token: 'abcd1234',
+                  refresh_token: 'efgh5678',
+                  developer_token: 'ijkl9101',
+                },
+                jobId: 2,
+                userId: 'u2',
+              },
+            },
+            {
+              destination: {
+                Config: {
+                  rudderAccountId: '258Yea7usSKNpbkIaesL9oJ9iYw',
+                  listId: '7090784486',
+                  customerId: '7693729833',
+                  loginCustomerId: '',
+                  subAccount: false,
+                  userSchema: ['email', 'phone', 'addressInfo'],
+                  isHashRequired: true,
+                  typeOfList: 'General',
+                },
+              },
+              message: {
+                action: 'delete',
+                context: {
+                  ip: '14.5.67.21',
+                  library: {
+                    name: 'http',
+                  },
+                },
+                recordId: '2',
+                rudderId: '2',
+                fields: {
+                  email: 'test@abc.com',
+                  phone: '@09876543210',
+                  firstName: 'test',
+                  lastName: 'rudderlabs',
+                  country: 'US',
+                  postalCode: '1245',
+                },
+                type: 'record',
+              },
+              metadata: {
+                secret: {
+                  access_token: 'abcd1234',
+                  refresh_token: 'efgh5678',
+                  developer_token: 'ijkl9101',
+                },
+                jobId: 3,
+                userId: 'u3',
+              },
+            },
+          ],
+          destType: 'google_adwords_remarketing_lists',
+        },
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: [
+            {
+              batchedRequest: [
+                {
+                  version: '1',
+                  type: 'REST',
+                  method: 'POST',
+                  endpoint:
+                    'https://googleads.googleapis.com/v15/customers/7693729833/offlineUserDataJobs',
```

Comment:
The current API version is [16](https://developers.google.com/google-ads/api/docs/release-notes#v1610_2024-04-24) can we update that?

Reply:
updated